dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -538,6 +547,50 @@ void MPPTask::runImpl()
     unregisterTask();
 }
 
+void MPPTask::reportStatus(const String & err_msg)
```

Comment:
It seems that warnings can also be passed through coodinator, ref https://github.com/pingcap/tiflash/issues/6358

Reply:
Or adding todo here?

Reply:
It's ok to pass warnings to coordinator, since the err_msg will take effect only when the root task returned error messages in tidb side.

Reply:
ReportMPPTaskStatus is introduced to solve the error and cancel-by-limit case, and it is not guaranteed tidb side will received all reports before mpp_gather closed. I prefer to add warnings in MppDataPacket, since warning message was not that important when error happened.

Reply:
I'm talking about the warnings in show warnings. I feel like adding a TODO or processing it in the same way as the summary.

Reply:
Yeah, I can add a TODO like "when MppDataPacket include warning messages info, support it in report status also"

Reply:
ok